recipes-kernel/linux/linux-yocto_%.bbappend

```diff
@@ -0,0 +1,11 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}:"
+
+SRC_URI += "file://meson64-kmeta;type=kmeta;destsuffix=meson64-kmeta"
+
+COMPATIBLE_MACHINE_khadas-vim3 = "khadas-vim3"
+
+KERNEL_DEVICETREE_khadas-vim3 = "amlogic/meson-g12b-a311d-khadas-vim3.dtb"
```

Comment:
Shouldn't this go into conf/machine/khadas-vim3.conf ?

Reply:
It is already there.  I will check to see if this line can safely be removed.